src/Enumerator.php

```diff
@@ -1,6 +1,6 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
Should we disabled the `declare_strict` fixer?

Reply:
Why?

Reply:
Just wondering!